<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Image_process</class>
 <widget class="QWidget" name="Image_process">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image_process</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 0, 0);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="pb_5">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 255, 127);</string>
     </property>
     <property name="text">
      <string>algoritm</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pb_4">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 255, 0);</string>
     </property>
     <property name="text">
      <string>rotate</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pb_3">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 255, 0);</string>
     </property>
     <property name="text">
      <string>image2</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pb_2">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 170, 0);</string>
     </property>
     <property name="text">
      <string>image1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pb_1">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 85, 0);</string>
     </property>
     <property name="text">
      <string>chess</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
